src/main/resources/config/liquibase/changelog/20190928155324_changelog.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet id="20190928155324" author="ungar">
+        <addColumn tableName="programming_exercise_details">
+            <column name="project_key" type="varchar(255)">
+            </column>
+        </addColumn>
+        <sql>
+            insert into programming_exercise_details (id, build_and_test_student_submissions_after_due_date, project_key)
```

Comment:
(Optional)

Reply:
This inserts new rows for all old programming exercises, which don't have an entry in the details table, yet. You are right, this is needed, so that in the next step all exercises have an entry that can be updated

Reply:
I added some comments to the migration script in order to clarify what it does